static/audiofeed.yaml

```
@@ -1,3 +1,30 @@
+  date: 2025-08-05T10:00
+  image: https://raw.githubusercontent.com/LucaNerlich/m10z/main/static/img/formate/cover/pixelplausch.jpg
+  description: |2-
+                  Mit der 4. Folge von Pixelplausch melden wir uns aus unserer Sommerpause zurück und sprechen mit André von Videospielgeschichten und unserem lieben M10Z-Kollegen Marcel aka kkuez über die folgenden Themen:
+
+                    - Diskussion über kleinere Videospiel-Messen als Alternative zur GamesCom.  
+                    - Lohnt es sich, die Entwicklungen bei AAA-Produktionen zu verfolgen und wie verteilt man die Zeit, die einem zum Computerspielen zur Verfügung steht?
+
+                  Beim zweiten Thema sind wir zeitweise etwas von der Ursprungsfrage abgekommen. Man kann bei solchen Diskussionen aber auch schnell in Kaninchenbauten fallen, wenn man nicht aufpasst. Wir fanden die Diskussion trotzdem sehr interessant und hoffen, dass es Euch beim Anhören genauso geht. 
+
+                  Und hier noch ein paar weiterführende Links zu den in der Folge angesprochenen Themen:
+                    - Link zur Polaris-Messe: <a href="www.https://www.polaris-con.de/">http://www.polaris-con.de/</a>  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)